x-pack/legacy/plugins/ml/public/data_frame/pages/job_management/components/job_list/expanded_row.tsx

```diff
@@ -65,6 +66,13 @@ export const ExpandedRow: SFC<Props> = ({ item, lastUpdate }) => {
       }),
       content: <TransformMessagesPane transformId={item.id} lastUpdate={lastUpdate} />,
     },
+    {
+      id: 'job-preview',
+      name: i18n.translate('xpack.ml.dataframe.jobsList.jobDetails.tabs.jobPreviewLabel', {
+        defaultMessage: 'Preview table',
```

Comment:
Nit: I don't think it needs to say `Preview table`, `Preview` should be fine?

Reply:
Updated: fd3c34164e5ead57a869c15608d253990f2a3f20